renderer/components/FormattedMarkdownMessage.js

```diff
@@ -39,8 +39,8 @@ class FormattedMarkdownMessage extends React.Component {
   render() {
     const { formatMessage } = this.context.intl
     const { id, defaultMessage, values, description } = this.props
-    const messageDescriptor = { id, defaultMessage, values, description }
-    const message = formatMessage(messageDescriptor)
+    const messageDescriptor = { id, defaultMessage, description }
+    const message = formatMessage(messageDescriptor, values)
```

Comment:
The correct syntax for `formatMessage` [as per docs](https://github.com/formatjs/react-intl/blob/master/docs/API.md#formatmessage).